src/components/viewSections/Terms.js

```diff
@@ -10,6 +10,24 @@ export default class Terms extends React.Component {
     record: PropTypes.shape({ customProperties: PropTypes.object }),
     recordType: PropTypes.string.isRequired,
     terms: PropTypes.arrayOf(PropTypes.object),
+    license: PropTypes.arrayOf(PropTypes.object),
+    amendment: PropTypes.arrayOf(PropTypes.object),
+  }
+
+  renderBadge = () => {
+    const { recordType } = this.props;
+    let customProperties = false;
+    if (recordType === 'license') {
+      customProperties = this.props.license.customProperties;
+    } else if (recordType === 'amendment') {
+      customProperties = this.props.amendment.customProperties;
+    }
+    if (customProperties) {
+      const count = Object.keys(customProperties).length;
```

Comment:
I think we want populated terms, not just all terms, something like:

```
      const count = Object.keys(customProperties).filter(cp => !!cp.value).length
```

(idk if that exactly will work)